packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_localizations_am.dart

```diff
@@ -102,7 +102,8 @@ class UbuntuProvisionLocalizationsAm extends UbuntuProvisionLocalizations {
   String get themePageTitle => 'Choose your theme';
 
   @override
-  String get themePageHeader => 'You can always change this later in the appearance settings.';
+  String get themePageHeader =>
```

Comment:
All these formatting changes should not be included (unless they were generated like that)